sklearn/utils/metaestimators.py

```diff
@@ -150,6 +150,12 @@ def available_if(check):
         a truthy value if the attribute is available, and either return False
         or raise an AttributeError if not available.
 
+    Returns
+    -------
+    callable
+        The callable should make the decorated method available if check returns
+        True, and make it unavailable otherwise.
```

Comment:
I do not think we say "should" when the behavior is clearly defined by the function. What do you think of this rewording?

```suggestion
        Callable makes the decorated method available if `check` returns
        a truthy value, otherwise the decorated method is unavailable.
```

Reply:
Thanks for the review. The rewording looks good!